Documentation/TeleportSystem/Overview.md

```diff
@@ -0,0 +1,88 @@
+# Teleport System
+
+The teleport system is a sub-system of the MRTK that handles teleporting the user when the
+application is using an opaque display. For AR experiences (like HoloLens), the teleportation
+system is not active. For immersive HMD experiences (OpenVR, WMR) the teleport system can
+be enabled.
+
+## Enabling and disabling
+
+The teleport system can be enabled or disabled by toggling the checkbox in its profile:
+
+![Teleport system profile](../../Documentation/Images/TeleportSystem/profile.png)
```

Comment:
do we actually need that image or are we ok with just describing the path of that checkbox / property?

we have a docs guideline for images to only include them if absolutely necessary. Someone would have to update them whenever that profile changes
https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#images--screenshots
(I know we're already showing profile screenshots in a lot of places)

Reply:
That's fair - I think in this case the profile is simple enough that text is sufficient. I've gotten rid of the image.